### DISCUSSION TITLE
Bug in documentation?

### DISCUSSION THREAD

**Discussion by rosakolejarz** [sqlalchemy/sqlalchemy#10184] [Usage Questions] [answered]
https://docs.sqlalchemy.org/en/20/tutorial/orm_related_objects.html#loading-relationships

![image](https://github.com/sqlalchemy/sqlalchemy/assets/60151899/d5b36b42-d46a-4026-93c1-bf3bb1a06f25)

> the [relationship()](https://docs.sqlalchemy.org/en/20/orm/relationship_api.html#sqlalchemy.orm.relationship) can determine unambiguously that there is a [one to many](https://docs.sqlalchemy.org/en/20/glossary.html#term-one-to-many) relationship from User.addresses to User;

are you sure "from User.addresses to User" ? If yes, can you explain?

**Comment by zzzeek** [comment, marked as answer]
hi -

this is obviously a typo.   feel free to send a PR in the future, will fix now

**Reply by zzzeek** [reply]
this will be corrected on the site in the next hour
